assets/components/countryGroupSwitcher/countryGroupSwitcher.scss

```diff
@@ -1,3 +1,5 @@
+@import '~stylesheets/gu-sass/gu-sass';
```

Comment:
this is not clear at first, it's done so `@extend .gu-content-margin` will work in these files - we want to rework the underlying grid moving forward, so this is an interim solution